docs/install-pmm/install-pmm-server/baremetal/docker/easy-install.md

```diff
@@ -10,12 +10,14 @@ Download and install PMM Server using `cURL` or `wget`:
 === "cURL"
 
     ```sh
+    export PMM_REPO=perconalab/pmm-server PMM_TAG=3.0.0-beta
     curl -fsSL https://www.percona.com/get/pmm | /bin/bash
```

Comment:
@catalinaadam we need to update this link as well

Reply:
let's use https://raw.githubusercontent.com/percona/pmm/refs/heads/v3/get-pmm.sh for now

Reply:
thanks @BupycHuk , I've updated all references to this links  + Docker tags